src/ansys/dpf/core/server_types.py

```diff
@@ -1229,11 +1275,18 @@ def __init__(
         docker_config: DockerConfig = RUNNING_DOCKER,
         use_pypim: bool = True,
         context: server_context.ServerContext = server_context.SERVER_CONTEXT,
+        grpc_mode: server_factory.GrpcMode = server_factory.DEFAULT_GRPC_MODE,
```

Comment:
@ansys-akarcher do we really need `mTLS` for InProcess? What's the potential risk? There is no gRPC connection here.